### PR TITLE
New config profile KAUST

### PR DESCRIPTION
**Pull request by husensofteng**
---
name: New Config for KAUST
about: A new cluster config for KAUST Ibex cluster
---

Please follow these steps before submitting your PR:

- [ ] If your PR is a work in progress, include `[WIP]` in its title
- [+] Your PR targets the `master` branch
- [ ] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [+] Add your custom config file to the `conf/` directory
- [+] Add your documentation file to the `docs/` directory
- [+] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [+] Add your custom profile to the `README.md` file in the top-level directory
- [+] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`
- [ ] OPTIONAL: Add your custom profile path and GitHub user name to `.github/CODEOWNERS` (`**/<custom-profile>** @<github-username>`)

The purpose of this new config is to help users on the KAUST Ibex cluster to run nf-core workflows smoothly. 

Thank you!

### CODE REVIEW

File: conf/kaust.config
Comment (jfy133) [inline conf/kaust.config:39]: Be wary of this, if users don't clean up their work directories manually and regularly, you may have very filled up hard drives very quickly 

Nf-core pipelines come with a debug profile that can turn this off if necessary

File: conf/kaust.config
Comment (jfy133) [inline conf/kaust.config:21]: Please double check this works and nextflow doesn't try to resolve the variable [rather than bash] (this can happen in some of the config sections)

Comment (jfy133) [reply conf/kaust.config:21]: And does the cache directory already exist for all users?

**Review by jfy133 — APPROVED** [review]

File: conf/kaust.config
Comment (husensofteng) [reply conf/kaust.config]: thanks for the note, I have now removed this flag. I thought to keep the decision to the user!

**Comment by jfy133** [comment]
Sorry @husensofteng I didn't see you were waiting for test execution approval. 

So you don't need to wait next time, feel free to join the nf-core GitHub org as described on https://nf-co.re/join

File: conf/kaust.config
Comment (husensofteng) [reply conf/kaust.config:21]: I have already tested this and it works :) 
yes, it will automatically create the cache directory if it doesn't exist

File: conf/kaust.config
Comment (jfy133) [reply conf/kaust.config]: It's better to keep it,= true actually,

Default is set to  false!

**Comment by husensofteng** [comment]
> Sorry @husensofteng I didn't see you were waiting for test execution approval.
> 
> So you don't need to wait next time, feel free to join the nf-core GitHub org as described on https://nf-co.re/join

thank you, I just sent a request!

File: conf/kaust.config
Comment (husensofteng) [inline conf/kaust.config:12]: @jfy133 the profile does not recognize this statement, do you you have any suggestion how to make it work? 
The purpose is to host a custom set of genome resources (similar to igenomes but locally maintained).

Comment (jfy133) [reply conf/kaust.config:12]: I think users would have to specify it with a custom config i.e. with `-c`... @maxulysse any ideas?

Comment (jfy133) [reply conf/kaust.config:12]: Unfortauntely this is breaking other pipeliens, so I'm commenting it out ofr now here: https://github.com/nf-core/configs/pull/810

**Pull request merged**
